pkg/controllers/managementuser/resourcequota/resource_quota_validate.go

```diff
@@ -3,8 +3,8 @@ package resourcequota
 import (
 	"fmt"
 
-	v1 "github.com/rancher/rancher/pkg/generated/norman/core/v1"
 	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	corew "github.com/rancher/wrangler/v3/pkg/generated/controllers/core/v1"
```

Comment:
Please cross-check with https://github.com/rancher/rancher/pull/52103.

I had a CI integration test failure when I attempted such a conversion.
See https://github.com/rancher/rancher/pull/52103#issuecomment-3350562756

At the moment that PR also contains wrangler conversions in sync, validate and reset.
Depending on which PR merges first the other will have to perform conflict resolution.
I honestly hope that this here merges first, because then my PR can be reduced to the core change, instead of adding technically unrelated code conversions.

Reply:
Happy to get this PR in first 😄
Regarding the integration test failure, is this an already existing tests or added as part of your PR? Note all the tests pass for this one.
Also, it may help testing both changes together if you create a new branch rebasing your changes on top of this one, even creating a draft PR with it to early check the CI results?

Reply:
Already existing test. Did not touch the integration tests in my PR. I am glad that it passes for this here.
New branch off this with mine top - Yes, that will be needed anyway, for me to resolve any merge conflicts.